packages/wallets/src/components/WalletGradientBackground/WalletGradientBackground.scss

```diff
@@ -707,6 +707,7 @@
                             rgba(0, 147, 147, 0.48) 100%
                         )
                         #ffffff;
+                    border-radius: 0.8rem;
```

Comment:
I forgot to include the border radius when I refactored the stylings for the gradient background here:
https://github.com/binary-com/deriv-app/pull/14660/files#diff-7d8067c135a62cf93e00fab6823868a5cac0d43ef32c21523f42ccff6206da5f